### ISSUE TITLE
Misleading comment in exercise 6

### ISSUE THREAD

**Issue by LostCrew** [yosuke-furukawa/tower-of-babel#36]
Line 13–15 are as follows:

```js
// This variable `c` should only be valid in this block.
var|let|const b = 8;
console.log(b); // 8
```

It is pretty clear that the comment should refer to variable `b` instead of `c` (previously declared).
Nice concept game and enjoyable implementation too.

**Comment by yosuke-furukawa** [comment]
Thank you. I will fix it.

**Comment by yosuke-furukawa** [comment]
Fixed in v0.4.2, Thank you.
